src/sentry/api/endpoints/user_notification_fine_tuning.py

```diff
@@ -131,7 +137,10 @@ def put(self, request, user, notification_type):
         # make sure that the ids we are going to update are a subset of the user's
         # list of orgs or projects
         if not ids_to_update.issubset(parent_ids):
-            return Response(status=status.HTTP_403_FORBIDDEN)
+            bad_ids = ids_to_update - parent_ids
+            return Response({
+                'detail': 'User does not belong to at least one requested project (ids: %s).' % bad_ids
```

Comment:
I think the verbiage for this can be improved. The user would need to belong to all the ids in `bad_ids`; not at least one id.

Reply:
> The user would need to belong to all the ids in bad_ids; not at least one id.

Yes, agreed. Which only proves your point:

> I think the verbiage for this can be improved

because what I meant was: "Of the id's included in the request, at least one is bad. Here are all of the bad ones: xxxxxxx," which obviously wasn't clear!

How about, "At least one of the requested projects is not available to this user, because the user does not belong to the necessary teams. (ids of unavailable projects: %s)?"

Reply:
@lobsterkatie yea looks good to me 👍